src/Monitor/resources/cm_new.md

```diff
@@ -0,0 +1,121 @@
+## Incorrect Cmdlets
+
+- Get-AzActivityLog
+    - DetailedOutput
+    - MaxRecord
+- Get-AzAlertRule
+    - DetailedOutput
+- Get-AzAutoscaleSetting
+    - DetailedOutput
+- Get-AzMetric
+    - AggregationType
+    - StartTime
+    - EndTime
+    - MetricFilter
+    - DetailedOutput
+- Get-AzMetricDefinition
+    - DetailedOutput
+- New-AzActionGroup
+    - ActionGroupId
+    - WebhookProperty
+- New-AzAlertRule
+    - DisableRule
+- New-AzLogProfile
+    - RetentionInDays
+- New-AzMetricAlert
+    - ActionGroup
+    - DisableRule
+- New-AzScheduledQueryRule
+    - Source
+    - Schedule
+    - Action
+    - AsJob
+- Remove-AzMetricAlert
+    - AsJob
+- Set-AzActionGroup
+    - Receiver
+    - DisableGroup
+    - InputObject
+- Set-AzActivityLogAlert
+    - Action
+    - DisableAlert
+    - InputObject
+- Set-AzAutoscaleSetting
+    - InputObject
+    - DisableSetting
+- Set-AzDiagnosticSetting
+    - InputObject
+    - Enabled
+    - Category
+    - MetricCategory
+    - Timegrain
+    - RetentionEnabled
+    - RetentionInDays
+- Set-AzScheduledQueryRule
+    - InputObject
+    - Source
+    - Schedule
+    - Action
+    - AsJob
+
+## Correct Cmdlets
+
+- Disable-AzActivityLogAlert
+- Enable-AzActivityLogAlert
+- Get-AzActionGroup
+- Get-AzActivityLogAlert
+- Get-AzDiagnosticSetting
+- Get-AzLogProfile
+- Get-AzMetricAlert
+- Get-AzScheduledQueryRule
+- Remove-AzActionGroup
+- Remove-AzActivityLogAlert
+- Remove-AzAlertRule
+- Remove-AzAutoscaleSetting
+- Remove-AzDiagnosticSetting
+- Remove-AzLogProfile
+- Remove-AzScheduledQueryRule
+- Update-AzScheduledQueryRule
+
+## New Cmdlets
+
+- Get-AzAlertRuleIncident
+- Get-AzDiagnosticSettingCategory
+- Get-AzEventCategory
+- Get-AzMetricAlertStatus
+- Get-AzMetricBaseline
+- Get-AzMetricNamespace
+- Get-AzTenantActivityLog
+- Get-AzVMInsightOnboardingStatus
+- Invoke-AzCalculateMetricBaseline
+- New-AzActivityLogAlert
+- New-AzAutoscaleSetting
+- New-AzDiagnosticSetting
+- Set-AzLogProfile
+- Update-AzAlertRule
+- Update-AzAutoscaleSetting
+- Update-AzLogProfile
+- Update-AzMetricAlert
+
+## Missing Cmdlets
```

Comment:
This is just the output of the compare_modules.ps1 tool. Do you think we should add the missing cmdlets to the release notes along side with all the supported ones?

Reply:
The only thing I woudl add is a note about why we are removing cmdlets in the release notes for each one - It looks like many of these may no longer be necessary because of inlining.  Basically we should make a distinction between cmdlets that are removed because they are not *needed* and cmdlets we plan to ship in a future release, if that makes sense.

Otherwise, this looks good.  Excellent work